packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -476,6 +477,12 @@ class SizeTransition extends AnimatedWidget {
   /// A value of 0.0 (the default) indicates the center for either [axis] value.
   final double axisAlignment;
 
+  /// The factor to multiply the cross axis size of the child by.
+  ///
+  /// If the value of [crossAxisSizeFactor] is less than one, the child will be
+  /// clipped in the appropriate axis.
+  final double? crossAxisSizeFactor;
```

Comment:
I wonder if this should have a different name since it is not directly the counter of `sizeFactor` since it is not an animation.

Reply:
Sure! I'm all in for a different name, I wasn't really happy with this one. What would you suggest?

Reply:
maybe `staticCrossAxisSizeFactor` or `fixedCrossAxisSizeFactor`? I think its important to make it clear that the widget does not listen to this value and animate on it unlike `sizeFactor`.

Reply:
I decided to go with `fixedCrossAxisSizeFactor` and I renamed it in [refactor: Rename parameter to fixedCrossAxisSizeFactor](https://github.com/flutter/flutter/pull/134659/commits/c4c1746cb2b757a666cc79a66961969b329175a4)